docs/general/changelog.md

```diff
@@ -15,6 +15,10 @@ ms.service: vs-appcenter
 This document lists new features as they're added to [Visual Studio App Center](https://appcenter.ms). These features will also live for a while, marked as [SHIPPED], in the [App Center Roadmap](~/general/roadmap.md).
 
 
+### **April 11, 2019**
+- **Build support for Xcode 10.2**
+    - Added support for building your apps with Xcode 10.2.
```

Comment:
Actually the Xcode 10.2 is available only for native.iOS, native.macOS, RN.iOS and is not available for Xamarin.iOS.

So probably we should reflect it to not confuse customers who will try to find Xcode 10.2 for Xamarin